scripts/analytics/institution_summary.py

```diff
@@ -29,48 +33,68 @@ def get_events(self, date):
 
         # Convert to a datetime at midnight for queries and the timestamp
         timestamp_datetime = datetime(date.year, date.month, date.day).replace(tzinfo=pytz.UTC)
-        query_datetime = timestamp_datetime + timedelta(1)
+        query_datetime = timestamp_datetime + timedelta(days=1)
+        daily_query = Q(created__gte=timestamp_datetime)
+        public_query = Q(is_public=True)
+        private_query = Q(is_public=False)
+
+        # `embargoed` used private status to determine embargoes, but old registrations could be private and unapproved registrations can also be private
+        # `embargoed_v2` uses future embargo end dates on root
+        embargo_v2_query = Q(root__embargo__end_date__gt=query_datetime)
 
         for institution in institutions:
-            node_query = (
-                Q(is_deleted=False) &
-                Q(created__lt=query_datetime)
-            )
+            node_qs = institution.nodes.filter(is_deleted=False, created__lt=query_datetime).exclude(type='osf.registration')
```

Comment:
@brianjgeiger This PR had some fun merge conflicts and in fixing them ended up with some refactoring to what I think is a more readable version. 
Projects and nodes now use a separate base qs that removes some of the need for node_query, etc. 

This turns something like:
```
'private': institution.nodes.filter(project_private_query).exclude(type='osf.registration').get_roots().count(),
```
into 
```
'private': node_qs.filter(private_query).get_roots().count(),
```
If you think this change is undesirable I can swap em back, but I think it makes processing the individual queries easier. 

The node_summary has good test coverage but there never was a institution_summary tests. To validate I compared the data from the same day before and after and all the numbers line up as expected.